src/persist/src/cfg.rs

```diff
@@ -74,7 +74,8 @@ impl BlobConfig {
                     .to_string();
                 let role_arn = query_params.remove("aws_role_arn").map(|x| x.into_owned());
```

Comment:
It occurs to me that either all of these parameters should have an “aws” prefix or none of them should. @danhhz will probably have opinions. I’d lean towards none of them having the prefix, since this is already and S3 source an “aws” is therefore implied. Fine to leave as is for now; let’s just check with Dan when he’s online!

Reply:
I sent #13669 to address!